### PR TITLE
Updated `hugo.toml` for 1.28 release

### PR DESCRIPTION
**Pull request by Rishit-dagli**
/area release-eng

### CODE REVIEW

File: hugo.toml
Comment (sftim) [inline hugo.toml:1]: You also need to drop this, I think:
```toml
[[params.versions]]
fullversion = "v1.23.17" # legacy; use {{< skew currentPatchVersion >}} instead
                         # retain until all localizations have migrated
version = "v1.23"
githubbranch = "v1.23.17"
docsbranch = "release-1.23"
url = "https://v1-23.docs.kubernetes.io"
```

Comment (Rishit-dagli) [reply hugo.toml:1]: Ah, yes

**Pull request merged**